saleor/webhook/transport/asynchronous/transport.py

```diff
@@ -133,7 +137,14 @@ def create_deliveries_for_subscriptions(
         )
 
     with allow_writer():
-        EventPayload.objects.bulk_create(event_payloads)
+        created_event_payloads = EventPayload.objects.bulk_create(event_payloads)
+        for event_payload, payload_data in zip(
+            created_event_payloads, event_payloads_data
+        ):
+            event_payload.payload_file.save(
+                f"payload-{event_payload.pk}-{event_payload.created_at}.json",
```

Comment:
I'm not sure about adding `created_at` to the file name. Why do we need it? I think we should remove it. `event_payload.pk` is unique and should be enough to identify the file.